spec/agent/client.yml

```diff
@@ -14,6 +14,5 @@ ec-config:
     cps: {EC_CPS}
     hca: "{EC_HCA}"
     lpt: "{EC_LPT}"
-    pxy: {EC_PXY}
```

Comment:
actually, put a double quote around may solve the issue. like this
```yaml
pxy: "{EC_PXY}"
```
when the ```conf.pxy``` is not present, the value will be replaced by a empty string, and help the agent to negate the proxy. ```NO_PROXY|PROXY``` will be incorporated in the next release.